AmilcarRosadoPat/src/main/java/com/applause/auto/pageframework/Chunks/LoginAuroraChunk.java

```diff
@@ -0,0 +1,91 @@
+package com.applause.auto.pageframework.Chunks;
+
+import java.lang.invoke.MethodHandles;
+
+import com.applause.auto.framework.pageframework.util.logger.LogController;
+import com.applause.auto.framework.pageframework.web.AbstractPage;
+import com.applause.auto.framework.pageframework.web.PageFactory;
+import com.applause.auto.framework.pageframework.web.WebElementLocator;
+import com.applause.auto.framework.pageframework.web.factory.WebDesktopImplementation;
+import com.applause.auto.framework.pageframework.web.factory.WebPhoneImplementation;
+import com.applause.auto.framework.pageframework.web.factory.WebTabletImplementation;
+import com.applause.auto.framework.pageframework.webcontrols.Button;
+import com.applause.auto.framework.pageframework.webcontrols.EditField;
+import com.applause.auto.pageframework.pages.HomePageAurora;
+
+@WebDesktopImplementation(DesktopAuroraChunk.class)
+@WebTabletImplementation(TabletAuroraChunk.class)
+@WebPhoneImplementation(MobileAuroraChunk.class)
+
+public class LoginAuroraChunk extends AbstractPage {
+
+	protected final static LogController LOGGER = new LogController(MethodHandles.lookup().getClass());
+
+	/*
+	 * Public Actions
+	 */
+
+	public void UserPassword(String User, String Password) {
+		LOGGER.info(String.format("Entering destination %s into search field", User));
+		getUserTextBox().setText(User);
+
+		LOGGER.info(String.format("Entering destination %s into search field", Password));
+		getPasswordTextBox().setText(Password);
+	}
+
+	public HomePageAurora IngresarButtom() {
```

Comment:
que hace `HomePageAurora ` ?

Reply:
Ahi en ese metodo solo retorno lo que es el objeto de HomePageAurora pero indica que es mi page principal en donde se encuentra mis objetos de la pagina

```suggestion
	public HomePageAurora IngresarButtom() {
```